<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Upsampling_para</class>
 <widget class="QWidget" name="Upsampling_para">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Upsample</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_Projection">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Optimize Projection</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="using_threshol_process">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>96</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Stop Threshold:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="apply_add_point">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Points</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>31</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Psi Sigma:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="sigma">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="value">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Add Point Num:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="add_num">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>119</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Threshold:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="threshold">
        <property name="decimals">
         <number>10</number>
        </property>
        <property name="minimum">
         <double>-0.999999000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>60</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Edge Senstivity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="edge_paramete">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>3</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>CGrid Radius;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="radius">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>0.000010000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="video_speed">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_70">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>100</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>speed</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="begin_index">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_play_video">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Play Video</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_69">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>122</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>end index</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_64">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>122</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>begin index</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="end_index">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_62">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>200</y>
       <width>122</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>snapshot resolution</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="wlop_snapshot_index">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="wlop_snapshot_resolution">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_63">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>230</y>
       <width>122</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>snapshot index</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
